common/common-api/src/main/java/org/ow2/proactive/utils/ObjectByteConverter.java

```diff
@@ -242,4 +255,15 @@ public static Serializable base64StringToSerializable(String input) throws IOExc
         return answer;
     }
 
+    public static Map<String, byte[]> mapOfSerializableToByteArray(Map<String, Serializable> input) throws IOException {
+        if (input == null) {
+            return null;
+        }
+        HashMap<String, byte[]> answer = new HashMap<>(input.size());
+        for (Map.Entry<String, Serializable> entry : input.entrySet()) {
+            answer.put(entry.getKey(), ObjectToByteConverter.ObjectStream.convert(entry.getValue()));
+        }
+        return answer;
```

Comment:
You can replace this by:
```java
return input.entrySet().stream.collect(Collectors.toMap(entry -> entry.getKey(), entry-> ObjectToByteConverter.ObjectStream.convert(entry.getValue())));
```

Reply:
You will probably have issues with the upper code due to the Thrown exception. I think it would be great to remove the exception for the ObjectToByteConverter.ObjectStream.convert(). It this cannot be done forget about my comment.